src/checkstyle/checkstyle.xml

```diff
@@ -150,16 +153,14 @@ page at http://checkstyle.sourceforge.net/config.html -->
         <module name="JavadocMethod">
             <property name="scope" value="protected"/>
             <property name="severity" value="error"/>
-            <property name="allowMissingJavadoc" value="true"/>
             <property name="allowMissingParamTags" value="true"/>
             <property name="allowMissingReturnTag" value="true"/>
-            <property name="allowMissingThrowsTags" value="true"/>
-            <property name="allowThrowsTagsForSubclasses" value="true"/>
-            <property name="allowUndeclaredRTE" value="true"/>
         </module>
 
         <!-- Check that paragraph tags are used correctly in Javadoc. -->
-        <module name="JavadocParagraph"/>
+        <!-- This module causes all the license comments to fail. -->
+        <!-- TODO: Either leave this commented out (or remove it), or put <p> tags in the license files -->
+        <!-- <module name="JavadocParagraph"/> -->
```

Comment:
The `JavadocParagraph` module checks all the license header comments.  Because they aren't "strict" html, it fails.  This can be fixed by either adding `<p/>` tags for each paragraph (which is probably not desired), or leaving this module out.

Maybe an upstream change can be made to allow the ability to ignore some Javadoc comments.

NOTE: the `SuppressionCommentFilter` module doesn't seem to effect this rule.